src/mainviews/create-job.tsx

```diff
@@ -101,10 +105,26 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
           newComputeType = envList[0].default_compute_type;
         }
 
+        const outputFormats = outputFormatsForEnvironment(
+          envList,
+          envList[0].name
+        )?.map(format => format.name);
+
         props.handleModelChange({
           ...props.model,
           environment: envList[0].name,
-          computeType: newComputeType
+          computeType: newComputeType,
+          outputFormats: outputFormats
+        });
+        environment = envList[0].name;
```

Comment:
This assignment is unnecessary since `environment` doesn't get used after this. Remove the declaration on line 94 and delete this line.